app/models/foreman_google/gce.rb

```diff
@@ -60,6 +60,7 @@ def disks
     # This should return interface compatible with Fog::Server
     # implemented by ForemanGoogle::Compute
     def find_vm_by_uuid(uuid)
+      GoogleCompute.new(service: client, zone: zone, identity: uuid).tap(&:reload)
```

Comment:
Can we use `uuid: uuid` instead of the `identity: uuid`?

Reply:
Just curios, why to do `.tap(&:reload)` instead of `.reload`?

Reply:
I'd rather keep both as they are. uuid because it's how compute_resources are calling it, identity because that is how Fog::Compute is calling it and here we are translating it.

Reply:
I've changed this and instance is now lazy loaded.